src/include/sof/math/exp_fcn.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+#ifndef __SOF_MATH_BASE_EXPONENTIAL_H__
+#define __SOF_MATH_BASE_EXPONENTIAL_H__
+
+/* define constants */
+#define BIT_MASK_Q62P2 4611686018427387904LL
+#define BIT_MASK_LOW_Q27P5 134217728ULL
+#define CONVERG_ERROR 28823037607936LL
+#define QUOTIENT_SCALE 17179869184LL
+#define TERMS_Q23P9 8388608LL
+#define NUM_Q14P18 16384LL
+
+/* Include Files */
```

Comment:
i dont think we need these section comments (same for line 20)

Reply:
Comments must always tell the reader something they do not already know. Of course different readers have different knowledge levels but I think everyone understands `#include`